.eslintrc.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  settings: {
+    react: {
+      version: "detect"
+    }
+  },
+  extends: ["formidable/configurations/es6-react", "prettier"],
+  rules: {
+    "react/sort-comp": "off",
+    "import/no-unresolved": [2, { ignore: ["victory*"] }],
+    "max-statements": 0,
+    complexity: ["error", { max: 16 }],
+    "no-magic-numbers": ["error", { ignore: [-1, 0, 0.5, 1, 2, 90, 180, 270, 360] }]
+  },
+  overrides: [
+    {
+      files: ["*.ts", "*.tsx"],
+      extends: [
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@typescript-eslint/recommended-requiring-type-checking"
+      ],
+      parser: "@typescript-eslint/parser",
+      parserOptions: {
+        project: "./tsconfig.json",
+        tsconfigRootDir: __dirname
+      },
+      plugins: ["@typescript-eslint"],
+      rules: {
+        "@typescript-eslint/no-explicit-any": "off",
+        "@typescript-eslint/explicit-function-return-type": "off",
+        "no-magic-numbers": "off",
+        "import/no-duplicates": "off",
+        "react/no-multi-comp": "off",
+        "react/sort-comp": "off"
+      }
+    }
+  ]
```

Comment:
Added an override so typescript files can be linted using their own rules.

Reply:
Also, I changed it to a `js` file so I could use `__dirname`, and super also, I had to add in the `complexity` and `react/sort-comp` rules so that eslint would pass with the exisiting `.js` files like it did before the package upgrades.